docs/semgrep-code/policies.md

```diff
@@ -163,17 +163,17 @@ To change assigned modes, select either the top **Matching Rules** checkbox to s
 If you have the **multiple policies** feature, you can customize the rules that run on specific repositories. Currently, this beta is not accepting new participants.
 :::
 
-The multiple policies feature enables users to customize the rules that run on specific repositories.
+The multiple policies feature enables users to customize the rules that run on specific repositories (projects). Users create different policies that projects can be assigned to.
```

Comment:
```suggestion
The multiple policies feature enables users to customize the rules that run on specific projects (repositories). Users create different policies that projects can be assigned to.
```

Since our preferred term here is "projects" and the "(repositories)" is just to help the user remember what they are.